mm/ktsan/stack_depot.c

```diff
@@ -3,40 +3,29 @@
 #include <linux/jhash.h>
 #include <linux/kernel.h>
 
-#define KT_STACK_HASH_SEED 0x9747b28c
-
 /* Only available during early boot. */
 void __init kt_stack_depot_init(kt_stack_depot_t *depot)
 {
 	unsigned i;
 
-	for (i = 0; i < KT_STACK_DEPOT_PARTS; i++) {
+	for (i = 0; i < KT_STACK_DEPOT_PARTS; i++)
 		depot->parts[i] = NULL;
-	}
 	kt_cache_init(&depot->stack_cache, KT_STACK_DEPOT_MEMORY_LIMIT, 1);
 	depot->stack_offset = 0;
 	kt_spin_init(&depot->lock);
 }
 
 /* Only available during early boot. */
-void __init kt_stack_depot_destroy(kt_stack_depot_t *depot)
+void kt_stack_depot_destroy(kt_stack_depot_t *depot)
```

Comment:
What is the intended usage scenario of this call during kernel loading?
I don't see any. If we call it, it should be during testing or something along these lines. We don't do testing during kernel loading.